tools/fastqc_dockerfile.yml

```diff
@@ -0,0 +1,30 @@
+class: DockerRequirement
+dockerPull: genomicpariscentre/fastqc
+dockerFile: |
+  ############################################################
+  # Dockerfile to build FASTQC container images
+  # Based on Ubuntu
+  ############################################################
+  
+  # Set the base image to Ubuntu 
+  FROM ubuntu:14.04 
+  # File Author / Maintainer 
```

Comment:
`fastqc` 11.4 is in Debian Jessie, and 11.5 in Debian testing: https://packages.debian.org/search?keywords=fastqc
If you really want to use an Ubuntu base image then look here for the available images: http://packages.ubuntu.com/search?keywords=fastqc

Either way the rest of the file reduces to `apt-get install fastqc`